app/src/main/java/swati4star/createpdf/util/PDFUtils.java

```diff
@@ -654,42 +654,43 @@ private boolean isInputValid(String path, String[] ranges) {
      * ERROR_INVALID_INPUT  if input is invalid like -3 or 3--4 or 3,,4
      */
     public static int checkRangeValidity(int numOfPages, String[] ranges) {
-        int startPage;
-        int endPage;
-
+        int startPage = -1;
+        int endPage = -1;
+        int returnValue = 0;
```

Comment:
change 0 to `NO_ERROR`

Reply:
Made the changes and the test case is also working by adding break statements after assignment to returnValue. 
The PR has been updated.